packages/pigeon/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 13.1.2
 
 * Adds compatibilty with `analyzer` 6.x.
+* [cpp] Fix C++ generator skipping FlutterError if no FlutterAPI is defined.
```

Comment:
This will need to be it's own version bump. With associated changes required (changelog, pubspec, and generator_tools)